comet/manager.py

```diff
@@ -109,18 +110,14 @@ def register_start(self, start_time, version, config=None, register_datasets=Fal
             )
         if not isinstance(start_time, datetime.datetime):
             raise ManagerError(
-                "start_time needs to be of type 'datetime.datetime' (is {}).".format(
-                    type(start_time).__name__
-                )
+                f"start_time needs to be of type 'datetime.datetime' (is {type(start_time).__name__})."
             )
         # TODO python2 support
-        if sys.version_info < (3, 0) and isinstance(version, unicode):
+        if sys.version_info < (3, 0) and isinstance(version, unicode):  # noqa: F821
             version = version.encode("utf-8")
```

Comment:
Either this should be deleted because, per the pyproject file, the minimum supported python version is 3.10, or we should actually support Python2 and the pyproject should reflect that.

I think the reason for python2 support is the use of the manager in the cal/flagging brokers, which are still running in Python2.  I don't know what the best solution is here.  Perhaps it's good enough to say those packages can use old versions of comet, if they need to be reinstalled.